nototools/sync_repos.py

```diff
@@ -84,17 +84,17 @@ def noto_checkout(
       failed_tags.append('%s: %s' % (r, t))
 
   if failed_tags:
-    print >> sys.stderr, 'failed to find:\n  %s' % '\n  '.join(failed_tags)
+    sys.stderr.write('failed to find:\n  %s' % '\n  \n'.join(failed_tags))
```

Comment:
The added newline is in the wrong place.

Reply:
Yes!